src/main/java/com/mitchej123/hodgepodge/asm/InventoryEffectRendererTransformer.java

```diff
@@ -0,0 +1,37 @@
+package com.mitchej123.hodgepodge.asm;
+
+import com.mitchej123.hodgepodge.asm.util.AbstractClassTransformer;
+import com.mitchej123.hodgepodge.asm.util.AbstractMethodTransformer;
+import cpw.mods.fml.relauncher.FMLLaunchHandler;
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.tree.*;
+
+public class InventoryEffectRendererTransformer extends AbstractClassTransformer {
+    public InventoryEffectRendererTransformer()
+    {
+        if (!FMLLaunchHandler.side().name().equals("SERVER")) {
```

Comment:
Why do you check if it's server side, when the class has @SideOnly(CLIENT), so it will never be loaded on server?

Reply:
_safety first_

Reply:
On a serious note, I'm not sure if the calling code actually checks if the class is loaded.